bucket/python.json

```diff
@@ -28,6 +45,19 @@
             "}"
         ]
     },
+    "pre_install": [
+        "'install-pep-514.reg', 'uninstall-pep-514.reg' | ForEach-Object {",
+        "    $py_root = \"$dir\".Replace('\\', '\\\\')",
+        "    $py_version = [regex]::match($version,'(\\d+.\\d+)').Groups[0].Value",
+        "    $content = Get-Content \"$dir\\$_\"",
+        "    $content = $content.Replace('$py_root', $py_root)",
+        "    $content = $content.Replace('$py_version', $py_version)",
+        "    if ($global) {",
+        "       $content = $content.Replace('HKEY_CURRENT_USER', 'HKEY_LOCAL_MACHINE')",
+        "    }",
+        "    Set-Content \"$dir\\$_\" $content",
```

Comment:
```suggestion
        "    Set-Content \"$dir\\$_\" $content -Encoding Ascii",
```

Reply:
Why ascii?

Powershell defaulted to exporting with utf-8 no bom and that correctly imported into regedit. If we switched to ascii, then what would happen if someone had unicode in their username (and scoop install path)?

regedit exports with utf-16le with BOM. The closest [matches I see](https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Management/Set-Content?view=powershell-7) are UTF8BOM and BigEndianUnicode.

I'd prefer to stick to the more common and functioning utf8 instead of something more esoteric.

Reply:
Powershell 6 and 7 default is UTF8 without BOM

Not powershell 5

Reply:
Why wouldn't `-Encoding UTF8` be sufficient?
(I'm using powershell 5.1 and utf8nobom was default.)

Reply:
I see. ps5 outputs bom for `-Encoding utf8`. That's fine. I changed it to that and tested that it correctly imports to regedit.